tslint.json

```diff
@@ -18,7 +18,8 @@
     "interface-name": [true, "never-prefix"],
     "space-before-function-paren": [true, { "anonymous": false }],
     "object-literal-sort-keys": false,
-    "object-literal-key-quotes": [true, "as-needed"]
+    "object-literal-key-quotes": [true, "as-needed"],
+    "only-arrow-functions": false
```

Comment:
Related to previous comment. In the test, you'd use a normal function, but in the module wrapper, and arrow function works fine, because there's no `this` needed.